vcd/config_test.go

```diff
@@ -626,6 +626,9 @@ func getConfigStruct(config string) TestConfig {
 		_ = os.Setenv("VCD_MAX_RETRY_TIMEOUT", newRetryTimeout)
 	}
 	if configStruct.Provider.SysOrg == "" {
+		if configStruct.VCD.Org == "" {
+			panic(fmt.Errorf("both 'provider.sysOrg' and 'vcd.org' are empty in the provided configuration: %s", config))
```

Comment:
Just a thought: maybe dump whole config on these errors to aid debugging? Of course, without sensitive information.